content/learn/getting-started/node-with-typescript.en.md

```diff
@@ -55,10 +55,14 @@ Now we can compile it to JavaScript using `tsc` command in the terminal. Let's d
 ```bash
 npx tsc example.ts
 ```
+> <a href="https://www.npmjs.com/package/npx">npx</a> here stands for Node Package Execute. This tool allows us to run TypeScript's compiler without having to install it globally.
```

Comment:
```suggestion
> [npx](https://www.npmjs.com/package/npx) here stands for Node Package Execute. This tool allows us to run TypeScript's compiler without installing it globally.
```

Does this work?

Reply:
It does!
![image](https://user-images.githubusercontent.com/84540554/199267492-337a3bbf-3d1a-4d31-8392-8d4956d3029d.png)